apps/web/src/lib/formatTime.ts

```diff
@@ -0,0 +1,7 @@
+import dayjs from 'dayjs';
+
+const formatTime = (date: Date | undefined): string => {
+  return date ? dayjs(date).format('hh:mm a - MM/DD/YY') : '';
```

Comment:
In the future, the format can be localized here, for example by getting the format string from i18next, instead of using hardcoded 'hh:mm a - MM/DD/YY'.